### DISCUSSION TITLE
How to deploy the MERN project?

### DISCUSSION THREAD

**Discussion by goldsunshines** [Finalgoal231/support#16] [Q&A] [answered]
There are 3 steps to deploy the MERN project.
Frontend deploy - Vercel
Backend deploy - I don't know well the best choice
DB deploy - Same, I don't know well

**Comment by noahstoner** [comment, marked as answer]
1. Set Up Your MERN Application Locally.
2. Create a Render Account.
3. Prepare Your Application for Deployment.
4. Set Up Your Database.
5. Configure Environment Variables on Render.
6. Deploy Your Backend.
7. Deploy Your Frontend.

Access Your Application.
